docs/src/main/paradox/index.md

```diff
@@ -3,9 +3,11 @@
 Akka is a toolkit and runtime for building highly concurrent, distributed, and fault-tolerant event-driven applications on the JVM. Akka can be used with both Java and Scala.
 This guide introduces Akka by describing the Scala version of the Hello World example. If you prefer to use Akka with Java, switch to the [Akka Quickstart with Java guide](https://developer.lightbend.com/guides/akka-quickstart-java/). 
 
-Actors are the unit of execution in Akka. The Actor model is an abstraction that makes it easier to write correct concurrent, parallel and distributed systems. The Hello World example illustrates Akka basics. Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
+Actors are the unit of execution in Akka. The Actor model is an abstraction that makes it easier to write correct concurrent, parallel and distributed systems. 
+The Hello World example illustrates Akka basics. Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. 
+This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
 
-After trying this example the comprehensive [Getting Started Guide](http://doc.akka.io/docs/akka/current/scala/guide/introduction.html) is a good next step to continue learning more about Akka.
+After trying this example the comprehensive [Getting Started Guide](http://doc.akka.io/docs/akka/2.6/scala/guide/introduction.html) is a good next step to continue learning more about Akka.
```

Comment:
link should be `typed/guide/`
(maybe the old is redirected)

Reply:
Yup it redirects